README.md

```diff
@@ -5,6 +5,8 @@ Python 3.3+'s [ipaddress](http://docs.python.org/dev/library/ipaddress) for Pyth
 
 This repository tracks the latest version from cpython, e.g. ipaddress from cpython 3.8 as of writing.
 
+This is the `ActiveState <https://activestate.com>`_ fork to provide CVE fixes for Python 2.7.
```

Comment:
Trailing `_` shows up on rendered README.

I think this is a rst thing? but just a `_` in md.